### PR TITLE
Add block_actions type support, close #4

### LINKED ISSUE #4: Action processing error
An error appears when the interactive button is pressed, the action remains unprocessed:

`
01:12:32 web.1  | W, [2020-10-28T01:12:32.771673 #12006]  WARN -- : Grape::Exceptions::ValidationErrors: payload[callback_id] is missing, payload[actions][0][value] is missing
`

### PR DESCRIPTION
**Pull request by CrazyOptimist**
Would be great if you review this and advice!
I got it working with my rails example.

### CODE REVIEW

**Comment by CrazyOptimist** [comment]
Very basic working example can be found [here](https://github.com/CrazyOptimist/slack-rails-bot-starter/tree/feature/support-block_actions)

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 #### 0.2.1 (Next)
 
+* Add support for block_actions interaction type - [@CrazyOptimist](https://github.com/CrazyOptimist).
```
Comment (dblock) [inline CHANGELOG.md:5]: Add a PR number in front:

```
[#6](https://github.com/slack-ruby/slack-ruby-bot-server-events/pull/6): Add ...
```

**Review by dblock — APPROVED** [review]
This looks good! Needs something in README to say that all these formats are supported and I'd be happy to merge.

File: CHANGELOG.md (hunk 1 above)
Comment (CrazyOptimist) [reply CHANGELOG.md]: Done
Thank you so much for quick response!

**Comment by dblock** [comment]
@CrazyOptimist want to help with co-maintainence of this gem? If yes, drop me a note to dblock[at]dblock[dot]org with your rubygems account name/email so I can add you. You can maybe [make the next release](https://github.com/slack-ruby/slack-ruby-bot-server-events/blob/master/RELEASING.md).

**Pull request merged**